packages/directory/addon/templates/components/dir-table.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#light-table table
+  height=height
+  responsive=true
+  as |t|
+}}
+  {{t.head
+    fixed=true
+  }}
+
+  {{#t.body as |body|}}
+    {{#if isLoading}}
+      {{#body.loader}}
+        Loading...
+      {{/body.loader}}
+    {{/if}}
+
+    {{#if table.isEmpty}}
+      {{#body.no-data}}
+        No stuff found.
```

Comment:
Should the message here be based on you are searching or not?

Reply:
We can add that in once the search component is merged

Reply:
Search has been merged.